src/app/campaign/form/campaign-form.component.html

```diff
@@ -1,43 +1,49 @@
 <mat-card>
   <form [formGroup]="campaignForm" ngNativeValidate>
-    <mat-form-field>
-      <mat-label>Owner</mat-label>
-      <mat-select formControlName="set_account_uri" required>
-        <mat-option *ngFor="let acct of accounts" [value]="acct.self_uri">{{ acct.name }}</mat-option>
-      </mat-select>
+    <mat-form-field appearance="outline">
+      <mat-label>Campaign Name</mat-label>
+      <input matInput placeholder="Campaign Name" formControlName="name" required />
     </mat-form-field>
 
-    <mat-form-field class="example-full-width">
-      <input matInput placeholder="Name" formControlName="name" required />
+    <mat-form-field appearance="outline">
```

Comment:
This isn't filtering and its printing the URL to the selected item and not the name of it.

See the filter autocomplete example: https://material.angular.io/components/autocomplete/examples